packages/sanity/src/_internal/cli/actions/manifest/extractManifestAction.ts

```diff
@@ -0,0 +1,183 @@
+import {createHash} from 'node:crypto'
+import {mkdir, writeFile} from 'node:fs/promises'
+import {dirname, join, resolve} from 'node:path'
+import {Worker} from 'node:worker_threads'
+
+import {type CliCommandArguments, type CliCommandContext} from '@sanity/cli'
+import {minutesToMilliseconds} from 'date-fns'
+import readPkgUp from 'read-pkg-up'
+
+import {
+  type CreateManifest,
+  type CreateWorkspaceManifest,
+  type ManifestWorkspaceFile,
+} from '../../../manifest/manifestTypes'
+import {type ExtractManifestWorkerData} from '../../threads/extractManifest'
+import {getTimer} from '../../util/timing'
+
+const MANIFEST_FILENAME = 'create-manifest.json'
+const SCHEMA_FILENAME_SUFFIX = '.create-schema.json'
+
+/** Escape-hatch env flags to change action behavior */
+const FEATURE_ENABLED_ENV_NAME = 'SANITY_CLI_EXTRACT_MANIFEST_ENABLED'
+const EXTRACT_MANIFEST_DISABLED = process.env[FEATURE_ENABLED_ENV_NAME] === 'false'
```

Comment:
Beware the inverted naming here (is it too confusing?)

EXTRACT_MANIFEST_**DISABLED**  = process.env[SANITY_CLI_EXTRACT_MANIFEST_**ENABLED**] === 'false'

So when SANITY_CLI_EXTRACT_MANIFEST_ENABLED is anything other than literal string 'false', extract is disabled.

Yeah.... Looking at it now, it IS too confusing. I think I was trying to avoid implied double negations (!!) in the code, but I really just moved that into the naming, which is worse.

Imma change this to:
```suggestion
const EXTRACT_MANIFEST_ENABLED = process.env[FEATURE_ENABLED_ENV_NAME] !== 'false'
```

Then use `!` at the callsite. I think thats much clearer, and not flipping DISABLED and ENABLED in the constant names.